MastodonSDK/Sources/MastodonLocalization/Resources/Base.lproj/Localizable.strings

```diff
@@ -502,7 +502,6 @@ uploaded to Mastodon.";
 "Scene.Settings.Section.Preference.DisableEmojiAnimation" = "Disable animated emojis";
 "Scene.Settings.Section.Preference.OpenLinksInMastodon" = "Open links in Mastodon";
 "Scene.Settings.Section.Preference.Title" = "Preferences";
-"Scene.Settings.Section.Preference.TrueBlackDarkMode" = "True black dark mode";
```

Comment:
Did you remove `true_black_dark_mode` from `Localization/app.json`, too?

Reply:
Yes, it's here: https://github.com/mastodon/mastodon-ios/pull/973/files#diff-527687b68684efcd790236be5e68527ce4037772106c748a11729b8538367c20L725